### PR TITLE
Support 2022.2

### PR DESCRIPTION
**Pull request by koxudaxi**
# Related Issues
https://github.com/koxudaxi/pydantic-pycharm-plugin/pull/492
https://github.com/koxudaxi/pydantic-pycharm-plugin/pull/494
https://github.com/koxudaxi/pydantic-pycharm-plugin/issues/516
https://github.com/koxudaxi/pydantic-pycharm-plugin/issues/511

### CODE REVIEW

**Comment by koxudaxi** [comment]
@chbndrhnns @KotlinIsland 
I have created the PR :)

File: gradle.properties [hunk 1]
```diff
@@ -8,22 +8,24 @@ pluginVersion = 0.3.12
...
+platformVersion = 222.3345-EAP-CANDIDATE-SNAPSHOT
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 platformPlugins = python-ce
 
 # Java language level used to compile sources and to generate the files for - Java 11 is required since 2020.3
 javaVersion = 11
+languageVersion = 1.7
+apiVersion = 1.6
```
Comment (KotlinIsland) [inline gradle.properties:25]: If we are targeting 222 as a minimum, then we should be using api 1.7 and java 11, I think we should a toolchain here like in the plugin template project

Comment (KotlinIsland) [reply gradle.properties:25]: https://github.com/JetBrains/intellij-platform-plugin-template/blob/f3250aeef4c68761bc7d9dbdd59396cf7260efe3/build.gradle.kts#L31

**Comment by koxudaxi** [comment]
I updated the config. But, Jacoco doesn't cover any test cases 😢

**Comment by KotlinIsland** [comment]
Oh sorry!, I made a mistake, 2022.2 has Java 17, not java 11. If we are targeting 222 as a minimum we should have Java 17

**Comment by koxudaxi** [comment]
I found the issue 😭 
https://github.com/JetBrains/gradle-intellij-plugin/issues/1065

**Comment by KotlinIsland** [comment]
MAybe [kotlinx-kover](Maybe we could use https://github.com/Kotlin/kotlinx-kover) could be used?

**Pull request merged**